waspc/src/Wasp/Generator/WebAppGenerator/AuthG.hs

```diff
@@ -50,5 +55,71 @@ genCreateAuthRequiredPage auth =
 genUseAuth :: Generator FileDraft
 genUseAuth = return $ C.mkTmplFd (C.asTmplFile [relfile|src/auth/useAuth.js|])
 
+genAuthForms :: AS.Auth.Auth -> Generator [FileDraft]
+genAuthForms auth =
+  sequence
+    [ genLoginForm auth,
+      genSignupForm auth,
+      genAuthForm auth,
+      genStitchesConfig,
+      genSocialIcons
+    ]
+
+genSocialIcons :: Generator FileDraft
+genSocialIcons =
+    copyTmplFile [relfile|auth/forms/SocialIcons.jsx|]
+  where
+    copyTmplFile = return . C.mkSrcTmplFd
+
+genStitchesConfig :: Generator FileDraft
+genStitchesConfig =
+    -- TODO(matija): where should I put this? Maybe keep it localy scopped in auth/forms for now?
+    copyTmplFile [relfile|stitches.config.js|]
+  where
+    -- TODO(matija): duplicated, extract
+    copyTmplFile = return . C.mkSrcTmplFd
+
+genAuthForm :: AS.Auth.Auth -> Generator FileDraft
+genAuthForm auth =
+  compileTmplToSamePath
+    [relfile|auth/forms/Auth.jsx|]
+    [ "onAuthSucceededRedirectTo" .= getOnAuthSucceededRedirectToOrDefault auth
+    , "isUsernameAndPasswordAuthEnabled" .= AS.Auth.isUsernameAndPasswordAuthEnabled auth
```

Comment:
A lefthand comma-er, I see :D

Reply:
always! That's the main feature of Haskell for me, never again those pesky commas that are left dangling :D